management-api/src/main/webapp/views/create-product.jsp

```diff
@@ -1,8 +1,16 @@
 <!DOCTYPE html>
 <%@ taglib prefix="spring" uri="http://www.springframework.org/tags"%>
```

Comment:
Do you need spring and jstl core libs at all?

Reply:
Fixed, removed